docs/examples/dali_plugin/README.md

```diff
@@ -0,0 +1,63 @@
+# DALI plugin with Triton
+
+This example shows how to use custom operations implemented for DALI
+(i.e. DALI plugins) with Triton. As an example, we use here a custom DALI
+operation that copies the input to the output. You can find this copy-plugin in
+[DALI documentation](https://docs.nvidia.com/deeplearning/dali/master-user-guide/docs/examples/custom_operations/custom_operator/create_a_custom_operator.html).
+
+### Step 1: Obtain the plugin lib
+
+Firstly, you need to code and build DALI plugin. This topic is
+out of scope of this tutorial. You can find all relevant info in
+[DALI documentation](https://docs.nvidia.com/deeplearning/dali/master-user-guide/docs/examples/custom_operations/custom_operator/create_a_custom_operator.html). All you need to do is to follow linked tutorial to obtain `.so` library with your custom plugin.
+
+### Step 2: Serialize DALI pipeline which uses your custom operation
+
+As always when using DALI with Triton, you need to serialize your DALI pipeline.
+Again, this topic is broadly discussed in the tutorial linked above. To summarize,
+you need to use `nvidia.dali.plugin_manager` to load your plugin and use it in
+your pipeline. `custom_copy_pipeline.py` file shows, how do to it:
+
+    plugin_manager.load_library('/path/to/libcustomcopy.so')
+    
+    @dali.pipeline_def(batch_size=3, num_threads=1, device_id=0)
+    def pipe():
+        data = dali.fn.external_source(device="cpu", name="DALI_INPUT_0")
+        cp = dali.fn.custom_copy(data)
+        return cp
+
+### Step 3: Inject plugin lib into tritonserver docker image
+
+The `.so` file you have built in the previous step contains the
+definition of your operation. Therefore, you need to inject it
+into your tritonserver image, so that DALI would be able to conduct
+this operation. There are multiple ways to achieve that, including
+mounting (`-v` option), copying (`COPY` directive or `docker cp` command),
+pick whichever suits you best.
+
+Here we will assume, that your `.so` library is located at
+`/models/libcustomcopy.so` path in your tritonserver docker image.
+
+### Step 4: Update your config file
+
+DALI Backend will be able to pick up your plugin library, provided
+that you'll point it out in model configuration. In your model repository
+please add a parameter to the DALI model, listing colon-separated plugin libraries:
+
+    parameters: [
+        {
+            key: "plugin_libs"
+            value: { string_value: "/path/to/libplugin1.so:/another/path/libplugin2.so" }
```

Comment:
I find it a bit confusing - previous step says that it assumes some location of the file and now you show _two_ plugins in different location.